README.md

```diff
@@ -1,5 +1,7 @@
 # CERF Anticipatory Action framework in Bangladesh
 
+## Background
+
 Building on the work by individual agencies, such as the IFRC, WFP, FAO and NGOs, the pilot will methodologically combine three components, scaling-up anticipatory humanitarian action in Bangladesh:
```

Comment:
Not necessary but if you have links to the work from the other agencies, it might be nice to link to them?

Reply:
@milanoleonardo is there specific work that we could link to here?